primo/data_parser/well_data.py

```diff
@@ -991,8 +990,6 @@ def compute_priority_scores(self):
         # Check if all the required columns for supported metrics are specified
         # If yes, register the name of the column containing the data in the
         # data_col_name attribute
-        # TODO: Combine the check_columns_available method with this method.
-        # TODO: Check fill data consistency for ann_gas_production and
         # ann_oil_production. See _categorize_gas_oil_wells method for details.
 
```

Comment:
You should take away the complete TODO statement (also line 993). Are we sure that we want to get rid of these TODO statements because of pylint? I don't want to get rid of anything important.